cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -57,6 +64,12 @@ protected void configure() {
         expose(LineageWriter.class);
         bind(DatasetFramework.class).to(LineageWriterDatasetFramework.class);
         expose(DatasetFramework.class);
+        if (kafkaRequired) {
```

Comment:
Its required here because the ``LineageWriterDatasetFramework`` needs it (it binds a ``BasicLineageWriter``, which needs ``BusinessMetadataStore``, which needs this).

However, we discussed at https://github.com/caskdata/cdap/pull/3918#discussion_r39352536 that all of these need to be refactored into their own module.